gulp.d/tasks/build.js

```diff
@@ -51,12 +51,7 @@ module.exports = (src, dest, preview) => () => {
       // NOTE concat already uses stat from newest combined file
       .pipe(concat('js/site.js')),
     vfs
-      .src('js/theme.js', { ...opts, sourcemaps })
-      .pipe(uglify())
-      // NOTE concat already uses stat from newest combined file
-      .pipe(concat('js/theme.js')),
```

Comment:
NO. 
theme.js cannot be concat into site.js because it has to be included in the header scripts  and not in the footer scripts. 
If you don't do that then you w'll have a white flash each time you navigate to a new page using dark mode. This behavior can only be reproduced on production, probably because of some lag that doesn't occurs whit localhost.